suggester/src/main/java/org/opengrok/suggest/Suggester.java

```diff
@@ -64,7 +65,7 @@
 
     private static final String PROJECTS_DISABLED_KEY = "";
 
-    private static final Logger logger = Logger.getLogger(Suggester.class.getName());
+    private static final Logger LOGGER = Logger.getLogger(Suggester.class.getName());
```

Comment:
We in our internal project always use `logger` so I googled a bit: https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case

However, I know that other modules use upper case so it's up to you to decide what to do.

Reply:
okay, seems like only constants should be upper case.